source/hic_hal/cypress/psoc5lp/PSoC5/USBFS_cdc.c

```diff
@@ -0,0 +1,1147 @@
+/***************************************************************************//**
+* \file USBFS_cdc.c
```

Comment:
The USB files for CDC and other classes should not be necessary if you're using the existing USB stack and are only providing the SIE/PHY driver.

Reply:
code has been redesigned to use the existing USB stack and provides only the SIE / PHY driver.